params/config.go

```diff
@@ -899,21 +899,26 @@ func (c *ChainConfig) IsCurie(num *big.Int) bool {
 	return isForked(c.CurieBlock, num)
 }
 
-// IsDarwin returns whether num is either equal to the Darwin fork block or greater.
+// IsDarwin returns whether time is either equal to the Darwin fork time or greater.
 func (c *ChainConfig) IsDarwin(now uint64) bool {
 	return isForkedTime(now, c.DarwinTime)
 }
 
-// IsDarwinV2 returns whether num is either equal to the DarwinV2 fork block or greater.
+// IsDarwinV2 returns whether time is either equal to the DarwinV2 fork time or greater.
 func (c *ChainConfig) IsDarwinV2(now uint64) bool {
 	return isForkedTime(now, c.DarwinV2Time)
 }
 
-// IsEuclid returns whether num is either equal to the Darwin fork block or greater.
+// IsEuclid returns whether time is either equal to the Darwin fork time or greater.
 func (c *ChainConfig) IsEuclid(now uint64) bool {
 	return isForkedTime(now, c.EuclidTime)
 }
 
+// IsEuclidV2 returns whether time is either equal to the EuclidV2 fork time or greater.
+func (c *ChainConfig) IsEuclidV2(now uint64) bool {
+	return false // TODO: add EuclidV2 fork time
```

Comment:
Let's add `EuclidV2Time` to `ChainConfig` in this PR.

Reply:
added in [f607c18](https://github.com/scroll-tech/go-ethereum/pull/1111/commits/f607c1896bfdabfcae9474ec0d60cee62f9dd83e).